web/src/features/charts/bar-breakdown/BarElectricityExchangeChart.tsx

```diff
@@ -0,0 +1,106 @@
+import { CountryFlag } from 'components/Flag';
+import HorizontalColorbar from 'components/legend/ColorBar';
+import { ScaleLinear } from 'd3-scale';
+import { useTranslation } from 'react-i18next';
+import { ZoneDetail, ZoneKey } from 'types';
+import { CarbonUnits } from 'utils/units';
+
+import { EXCHANGE_PADDING } from './constants';
+import Axis from './elements/Axis';
+import CapacityLegend from './elements/CapacityLegend';
+import HorizontalBar from './elements/HorizontalBar';
+import Row from './elements/Row';
+import { ExchangeDataType } from './utils';
+
+export default function BarElectricityExchangeChart({
+  height,
+  width,
+  data,
+  exchangeData,
+  powerScale,
+  co2ColorScale,
+  graphUnit,
+  formatTick,
+  onExchangeRowMouseOut,
+  onExchangeRowMouseOver,
+}: {
+  height: number;
+  width: number;
+  data: ZoneDetail;
+  exchangeData: ExchangeDataType[];
+  powerScale: ScaleLinear<number, number, never>;
+  co2ColorScale: ScaleLinear<string, string, string>;
+  graphUnit: string;
+  formatTick: (t: number) => string | number;
+  onExchangeRowMouseOut: () => void;
+  onExchangeRowMouseOver: (
+    rowKey: ZoneKey,
+    data: ZoneDetail,
+    event: React.MouseEvent<SVGPathElement, MouseEvent>
+  ) => void;
+}) {
+  const { t } = useTranslation();
+
+  if (!exchangeData || exchangeData.length === 0) {
+    return null;
+  }
+
+  return (
+    <>
+      <div>
+        <CapacityLegend>
+          {t('country-panel.graph-legends.exchange-capacity')} ({graphUnit})
+        </CapacityLegend>
+      </div>
```

Comment:
Is this div needed?
Otherwise it would be good to remove it to keep the DOM count as small as possible.